tracer/build/_build/docker/serverless.lambda.dockerfile

```diff
@@ -5,14 +5,12 @@ RUN mkdir -p /var/log/datadog/dotnet && \
     chmod a+rwx /var/log/datadog/dotnet
 
 # Add Tracer
-COPY ./serverlessArtifacts /opt/datadog
+COPY ./test/test-applications/integrations/Samples.AWS.Lambda/bin/artifacts/monitoring-home /opt/datadog
```

Comment:
Just a thought, even though it helps to centralize the dockerfiles, we could reduce the size of the Docker context by moving this dockerfile over to `test/test-applications/integrations/Samples.AWS.Lambda` since those are all the files copied. Or I guess we could keep the dockerfile here, make the paths relative to the application directory, and then set the Docker context to the application directory 🤔 

Figured I'd throw this thought process out there since the purpose of this PR is to make improvements!

Reply:
You know, I considered this, but I only considered making the context be the docker folder, like we do elsewhere, but that doesn't work. Making the context be the sample directory would solve it nicely I think, thanks!